google-cloud-clients/google-cloud-logging/src/test/java/com/google/cloud/logging/v2/testing/LocalLoggingImpl.java

```diff
@@ -61,7 +61,7 @@ public void listLogEntries(
       ListLogEntriesRequest request, StreamObserver<ListLogEntriesResponse> responseObserver) {
     List<LogEntry> entries = new ArrayList<>();
 
-    for (ByteString proj : request.getProjectIdsList().asByteStringList()) {
+    for (ByteString proj : request.getResourceNamesList().asByteStringList()) {
```

Comment:
The former returns something like `{"[PROJECT_ID]" , ... }`  but 

what does getResourceNamesList() returns same value or with prefix type like `{"projects/[PROJECT_ID]", ...}`?